purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -354,25 +354,34 @@ class Purchases internal constructor(
         }
     }
 
+    /**
+     * Gets the StoreProduct(s) for the given list of product ids for all product types.
+     * @param [productIds] List of productIds
+     * @param [callback] Response callback
+     */
+    fun getProducts(
+        productIds: List<String>,
+        callback: GetStoreProductsCallback
+    ) {
+        getProducts(productIds, null, callback)
+    }
+
     /**
      * Gets the StoreProduct(s) for the given list of product ids.
```

Comment:
nit: maybe something like `Gets the StoreProduct(s) for the given list of product ids of type [type]`